backend/src/main/java/com/bakdata/conquery/models/messages/network/specific/RemoveSlave.java

```diff
@@ -0,0 +1,29 @@
+package com.bakdata.conquery.models.messages.network.specific;
+
+import com.bakdata.conquery.io.cps.CPSType;
+import com.bakdata.conquery.models.messages.network.MasterMessage;
+import com.bakdata.conquery.models.messages.network.NetworkMessage;
+import com.bakdata.conquery.models.messages.network.NetworkMessageContext;
+import com.fasterxml.jackson.annotation.JsonCreator;
+import lombok.Getter;
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+
+/**
+ * Message that allows salves to safely disconnect from Master node.
+ *
+ * @deprecated Doesn't do much at this moment (and is not stable at this moment), as the data is distributed ahead of time, but it can later be used.
+ */
+@Deprecated
+@CPSType(id = "REMOVE_SLAVE", base = NetworkMessage.class)
+@RequiredArgsConstructor(onConstructor_ = @JsonCreator)
+@Getter
+@Slf4j
+public class RemoveSlave extends MasterMessage {
+
```

Comment:
Shouldn't this be send in the shutdown sequence of  a slave?

Reply:
Ich habs mal nicht mit reingemacht, weil das einfach noch kein definiertes Verhalten hat. Da müssen wir mMn erstmal einen sinnvollen Umgang mit finden bevor wir das einfach einbauen.